app/assets/javascripts/app/manage_orders.js

```diff
@@ -228,10 +228,23 @@ class OrderDetailManagement {
 }
 OrderDetailManagement.initClass();
 
+function repricingButtonListener(orderDetailManagement) {
+  const repricingButton = document.querySelector(".js--recalculate-pricing");
+
+  if (repricingButton) {
+    repricingButton.addEventListener('click', () => orderDetailManagement.updatePricing());
+  }
+}
+
```

Comment:
I think think this should happen as part of `initPriceUpdating()`, since that is where the constructor class assigns other elements event listeners using `this.updatePricing()`.  You may need to adjust things a bit:
```
    const repricingButton = this.$element.find(".js--recalculate-pricing");

    if (repricingButton) {
      repricingButton.bind("click", () => this.updatePricing());
    }
```